docs/modules/ROOT/pages/quarkus-artemis-ra.adoc

```diff
@@ -43,7 +50,9 @@ Once the BOM is added, simply add the `io.quarkiverse.artemis:quarkus-artemis-ra
 </dependency>
 ----
 
-We recommend to set properties `quarkus.version` and `artemis.version` to the versions you are using/need to use. We also recommend to align `artemis.version` with the artemis server version used.
+We recommend to set properties `quarkus.version` and `artemis.version` to the versions you are using/need to use.
```

Comment:
Does this actually work? Have we tested this?

Reply:
Yes, I tested this with the integration test. Was planing on to do this with an example project as well.

Reply:
Okay. I'd say let's see what your example project says.

Reply:
Artemis bom import needs to precede the Quarkus one, but other than that it works. I also checked native image. This is the dependency:tree diff of an example file:

```diff
@@ -106,8 +106,8 @@
    +- io.quarkiverse.artemis:quarkus-artemis-core:jar:999-SNAPSHOT:compile
    |  +- io.quarkus:quarkus-netty:jar:3.17.5:compile
    |  |  \- io.netty:netty-codec:jar:4.1.115.Final:compile
-   |  \- org.apache.activemq:artemis-core-client:jar:2.38.0:compile
-   |     +- org.jgroups:jgroups:jar:5.3.13.Final:compile
+   |  \- org.apache.activemq:artemis-core-client:jar:2.37.0:compile
+   |     +- org.jgroups:jgroups:jar:5.3.10.Final:compile
    |     +- io.netty:netty-transport-native-epoll:jar:linux-x86_64:4.1.115.Final:compile
    |     +- io.netty:netty-transport-classes-epoll:jar:4.1.115.Final:compile
    |     +- io.netty:netty-transport-native-kqueue:jar:osx-x86_64:4.1.115.Final:compile
@@ -117,9 +117,9 @@
    |        +- org.bouncycastle:bcprov-jdk18on:jar:1.79:compile
    |        \- org.bouncycastle:bcpkix-jdk18on:jar:1.79:compile
    |           \- org.bouncycastle:bcutil-jdk18on:jar:1.79:compile
-   \- org.apache.activemq:artemis-jakarta-client:jar:2.38.0:compile
-      +- org.apache.activemq:artemis-commons:jar:2.38.0:compile
+   \- org.apache.activemq:artemis-jakarta-client:jar:2.37.0:compile
+      +- org.apache.activemq:artemis-commons:jar:2.37.0:compile
       |  \- commons-beanutils:commons-beanutils:jar:1.9.4:compile
       |     \- commons-collections:commons-collections:jar:3.2.2:compile
-      +- org.apache.activemq:artemis-selector:jar:2.38.0:compile
+      +- org.apache.activemq:artemis-selector:jar:2.37.0:compile
       \- jakarta.jms:jakarta.jms-api:jar:3.1.0:compile
```

and the native image produces `2.37.0` via `VersionLoader.getVersion().getFullVersion()`